### PR TITLE
[llvm-objcopy][test] Use llvm-readelf instead for clearer visualization

### PR DESCRIPTION
**Pull request by kongy**

### CODE REVIEW

File: llvm/test/tools/llvm-objcopy/ELF/prefix-symbols.test [hunk 1]
```diff
@@ -40,51 +40,11 @@ Symbols:
...
-# COMMON-NEXT:    Value: 0x0
-# COMMON-NEXT:    Size: 0
-# COMMON-NEXT:    Binding: Global
-# COMMON-NEXT:    Type: None
-# COMMON-NEXT:    Other: 0
-# COMMON-NEXT:    Section: Undefined
-# COMMON-NEXT:  }
-# COMMON-NEXT:]
+#COMMON:      Symbol table '.symtab' contains 5 entries:
+#COMMON-NEXT:  Num: Value Size Type Bind Vis Ndx Name
```
Comment (MaskRay) [inline llvm/test/tools/llvm-objcopy/ELF/prefix-symbols.test:44]: There is usually a space after `#`

File: llvm/test/tools/llvm-objcopy/ELF/prefix-symbols.test [hunk 2]
```diff
@@ -40,51 +40,11 @@ Symbols:
...
+#COMMON-NEXT:   0: {{.*}} 0 NOTYPE  LOCAL  {{.*}}
+#COMMON-NEXT:   1: {{.*}} 0 SECTION LOCAL  {{.*}} foo
```
Comment (MaskRay) [inline llvm/test/tools/llvm-objcopy/ELF/prefix-symbols.test:46]: For defined non-ABS symbols, `[[#]]` (which matches a decimal) is slightly better than `{{.*}}`

**Review by MaskRay — APPROVED** [review]

File: llvm/test/tools/llvm-objcopy/ELF/prefix-symbols.test (hunk 1 above)
Comment (kongy) [reply llvm/test/tools/llvm-objcopy/ELF/prefix-symbols.test:44]: Done.

File: llvm/test/tools/llvm-objcopy/ELF/prefix-symbols.test (hunk 2 above)
Comment (kongy) [reply llvm/test/tools/llvm-objcopy/ELF/prefix-symbols.test:46]: Done.

**Pull request merged**